contracts/PushComm/PushCommStorageV2.sol

```diff
@@ -4,7 +4,6 @@ import { CommTypes } from "../libraries/DataTypes.sol";
 
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "../interfaces/wormhole/INttManager.sol";
-import "../interfaces/wormhole/ITransceiver.sol";
```

Comment:
Just double-checking - Is ITransceiver not be used any more? @Zartaj0

Reply:
Initially, we had two transceiver variables. 
 `ITransceiver public TRANSCEIVER ` and  `IWormholeTransceiver public WORMHOLE_TRANSCEIVER`
This was removed [here](https://github.com/push-protocol/push-smart-contracts/commit/34a2bfde0d248671545b3782e7d55c18ac1e89e7) in this [PR](https://github.com/push-protocol/push-smart-contracts/pull/390)

Now `IWormholeTransceiver` inherits the `ITransceiver`